test/helpers/handshake.c

```diff
@@ -278,9 +278,12 @@ static int server_ocsp_cb(SSL *s, void *arg)
      * For the purposes of testing we just send back a dummy OCSP response
      */
     *resp = *(unsigned char *)arg;
-    if (!SSL_set_tlsext_status_ocsp_resp(s, resp, 1))
+    if (!SSL_set_tlsext_status_ocsp_resp(s, resp, 1)) {
+        OPENSSL_free(resp);
```

Comment:
I'm not sure why this was flagged.  The call to `SSL_set_tlsext_status_ocsp_resp()` cannot fail.

Reply:
The source code is the document :)  The call always returns 1.

Reply:
I think we should assume that it can fail. If not now, maybe in the future. We already check the return value anyway.